androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -57,6 +57,8 @@ fun EnhancedVideoPlayer(
     enableImmersiveMode: Boolean = true,
     playImmediately: Boolean = true,
     soundOff: Boolean = true,
+    hideControls: Boolean = false,
+    disableGestures: Boolean = false,
```

Comment:
I think it's not good to expose this `disableGestures` flag because `disableGestures` and hideControls mean the same thing, why would we want to disable the gestures and still show the controls? I think we should use hideControls to disable pointerEvents.

another point is `disableGestures` is only disabling the forward/rewind event if this is desired behavior change the name of the flag to `disableSeek`

Reply:
`disableGestures` means you want to disable all gestures like drag/drop, zoom, tap , double tap. [This link](https://developer.android.com/jetpack/compose/touch-input/pointer-input/understand-gestures#definitions) might help you understand the jetpack api nomenclature